src/components/CardSlide/CardSlide.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+import styles from './cardslide.module.css';
+
+type DataCard = {
+  data: CardProps; // type CarProps is in the file "index.d.ts" because it's needed in other file
+}
+
+// this card adapts to all that we need in various components
+const CardSlide: React.FC<DataCard> = ({ data }) => {
+  const {
+    image,
+    title,
+    description,
+    buttonArrow = false,
+    buttonMore = false
+  } = data;
+
+  return (
+    <div className={styles.card}>
+      <Image
+        width={300}
+        height={300}
+        className={`${styles.card__image} ${description && styles['card__image--with-description']}`}
```

Comment:
```suggestion
        className={`${styles.card__image} ${description && styles['card__image--with-description']}`}
```
Cards <b>could</b> have a function (with useEffect) `calculateCardMargin(numCards)` that calculates the distance between them.